app/models/volunteer.rb

```diff
@@ -73,12 +73,11 @@ def self.state_collection
     STATES.map(&:to_sym)
   end
 
-  DURATIONS = [:short, :long].freeze
-  REGIONS = [:city, :region, :canton].freeze
-  SINGLE_ACCOMPANIMENTS = [:man, :woman, :family, :kid].freeze
-  GROUP_ACCOMPANIMENTS = [:sport, :creative, :music, :culture, :training, :german_course].freeze
+  SINGLE_ACCOMPANIMENTS = [:man, :woman, :family, :kid, :unaccompanied].freeze
+  GROUP_ACCOMPANIMENTS = [:sport, :creative, :music, :culture, :training, :german_course,
+                          :dancing, :health, :cooking, :excursions, :women, :teenagers,
+                          :children, :other_offer].freeze
   REJECTIONS = [:us, :her, :other].freeze
-  TARGET_GROUPS = [:adults, :teenagers, :children].freeze
   GENDERS = [:female, :male].freeze
```

Comment:
just stumbling upon this, regarding my story from last week. I thought I changed all `genders` to `salutation(mrs/mr)`. is this constant used in matching? or even not used anymore?